app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def bootstrap_class_for(type)
+    { success: 'alert-success',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.